eval/http.go

```diff
@@ -78,7 +80,41 @@ func ApplyRequestContext(ctx context.Context, body hcl.Body, req *http.Request)
 		httpCtx = c.eval
 	}
 
-	content, _, _ := body.PartialContent(meta.AttributesSchema)
+	type Inline struct {
```

Comment:
This is unfortunately the wrong place to apply the `request` context options.

This method gets called from endpoint, proxy and backend evaluations. In this case I could write `body` or  `method` within a backend block and this would override the request one. This attributes should apply to `request` only.

This attribute evaluation should take place in `producer/request.go`.